### ISSUE TITLE
Update example about Conan as Python package manager

### ISSUE THREAD

**Issue by jgsogo** [conan-io/docs#1524]
https://docs.conan.io/en/latest/howtos/other_languages_package_manager/python.html

Here we are using Python 2

**Comment by memsharded** [comment]
Yes, I think it can and should be changed, together with the examples, that are still pointing to my personal repo. But it doesn't necessarily needs to be done in 1.22, it is lower priority, not a big issue that it is a bit outdated using Python2, so if no time for it, it can wait for later releases.
